requirements/tools/Dockerfile

```diff
@@ -9,6 +9,9 @@
 #
 #
 ARG PYTHON_VERSION="3.11.9"
+ARG UV_VERSION="0.4"
+FROM ghcr.io/astral-sh/uv:${UV_VERSION} AS uv_build
```

Comment:
interesting how they use this base image to "mount their executables" into another image.

Reply:
also I learned that ARG in Dockefile on top is global, and only "local" if you put it below